2022q4/xfce.adoc

```diff
@@ -4,8 +4,9 @@ Links: +
 link:https://xfce.org/about/news/?post=1671062400[Xfce 4.18 Upstream Release Announcement] URL: link:https://xfce.org/about/news/?post=1671062400[https://xfce.org/about/news/?post=1671062400] +
 link:https://www.freshports.org/x11-wm/xfce4[Xfce meta-port on FreshPorts] URL: link:https://www.freshports.org/x11-wm/xfce4[https://www.freshports.org/x11-wm/xfce4]
 
-Contact: Xfce team <xfce@FreeBSD.org>
-Contact: Guido Falsi <madpilot@FreeBSD.org>
+Contacts: +
+Xfce team <xfce@FreeBSD.org> +
+Guido Falsi <madpilot@FreeBSD.org>
```

Comment:
Please revert this change. As you can see in other reports, we usually write contacts in the form that was originally proposed.

It might be a good idea to improve how we manage links and contacts and we will probably do it at some point, but as you know we are already busy improving other aspects of our workflow and I do not think we have the time to deal with this too for this quarter.

Reply:
Reverted with [`44d75b7` (#575)](https://github.com/freebsd/freebsd-quarterly/pull/575/commits/44d75b79a3374d728a2120999adb79afdb5f2319). The result, previewed: 

> Contact: Xfce team <[xfce@FreeBSD.org](mailto:xfce@FreeBSD.org)> Contact: Guido Falsi <[madpilot@FreeBSD.org](mailto:madpilot@FreeBSD.org)>

– multiple contacts on a single line. Please, was this the intention?

Reply:
> … multiple contacts on a single line. Please, was this the intention?

Resolved with https://github.com/freebsd/freebsd-quarterly/commit/94489ef7562bd54ef71f098067be3d83b82b46fa?diff=unified#diff-8bd6d75de486569e8357dec0b6ad372cc892501eba032ca9c485a8cb276b3421